### PR TITLE
Implemeted

### PR DESCRIPTION
**Pull request by Himanshusingh770**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@headlessui/react": "^2.1.10",
+    "@heroicons/react": "^2.1.5",
```
Comment (PiyushMangla) [inline package.json]: why use 2 different library for icons

File: src/Components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import { FaUserCircle, FaBars } from 'react-icons/fa';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { logout } from '../slices/authSlice';
+import navbarData from '../utils/navbarData.json';
+import { ADMIN_ID } from '../utils/constantData';
+
+const Navbar = () => {
+  const [showLogoutDropdown, setShowLogoutDropdown] = useState(false);
+  const [showNavbarMenu, setShowNavbarMenu] = useState(false);
```
Comment (PiyushMangla) [inline src/Components/Navbar.jsx:11]: why?

File: src/Components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { FaUserCircle, FaBars } from 'react-icons/fa';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { logout } from '../slices/authSlice';
+import navbarData from '../utils/navbarData.json';
+import { ADMIN_ID } from '../utils/constantData';
```
Comment (PiyushMangla) [inline src/Components/Navbar.jsx]: HR too

File: src/Components/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+                    {item.name}
+                  </NavLink>
+                </li>
+              ))}
+          </ul>
+
+          {/* Horizontal Navbar for Desktop */}
+          <ul className="hidden md:flex md:space-x-6 md:bg-transparent">
+            <li>
+              <NavLink
```
Comment (yuvrajKaexpertise) [inline src/Components/Navbar.jsx]: remove this navlink use it from navbarData @Himanshusingh770

**Review by yuvrajKaexpertise — CHANGES_REQUESTED** [review]

File: src/Components/Navbar.jsx (hunk 4 above)
Comment (Himanshusingh770) [reply src/Components/Navbar.jsx]: ok sir

**Pull request merged**